Probesys/bileto
# Tec/refactor loading suborgas

## Related issue(s)



https://github.com/Probesys/bileto/issues/211
https://github.com/Probesys/bileto/issues/181

## Changes



- remove the code about setting parent organizations
- extract the logic to organize the organizations in tree in the `OrganizationSorter#asTree` method
- remove the method `OrganizationRepository#findAllAsTree`
- change the method `OrganizationRepository#findOneByAsTree` by `OrganizationRepository#findWithSubOrganizations` which load the requested organizations + their sub-organizations

## How to test manually



- Go to the organizations page, check the organizations are displayed as before
- Go to the new sub-organization page, check the organizations are displayed as before
- Go to the new authorization page, check the organizations are displayed as before

## Checklist
